app/lib/realm.js

```diff
@@ -1,15 +1,39 @@
 import Realm from 'realm';
-
 // import { AsyncStorage } from 'react-native';
-// Realm.clearTestState();
-// AsyncStorage.clear();
+// import RocketChat from './rocketchat';
```

Comment:
Add this line. 
```
// Realm.clearTestState();
```

It's useful during development and it's deleted in build release anyway.